code/modules/events/disease_outbreak.dm

```diff
@@ -18,6 +21,10 @@
 
 /datum/event/disease_outbreak/announce()
 	GLOB.event_announcement.Announce("Confirmed outbreak of level 7 major viral biohazard aboard [station_name()]. All personnel must contain the outbreak.", "Biohazard Alert", new_sound = 'sound/AI/outbreak7.ogg')
+	for(var/p in GLOB.dead_mob_list)
+		var/mob/M = p
+		if(isobserver(M) || M.stat == DEAD)
```

Comment:
Should always be the case right? Either a ghost or a dead mob

Reply:
In theory it should yeah. Some other ghost messages I saw had these sanity checks as well, such as the message shown when a player dies, so I just included them here to be safe. Think I should remove them?

Reply:
I see it in the code yeah. Tons of checks if it's a `new_player` or dead etc....
I just looked at the code where things are added and removed to `dead_mob_list` and it works as it says in the comment. Only gets added when something dies or is dead from the start and gets removed when it becomes alive again.
`new_player` mobs being the exception since they don't call their parents `Initialize`
So it should be fine to remove this. If there are bugs they better get solved than to add weird checks to lists that should not need those checks